### PR TITLE
Release 4.1.2

### PR DESCRIPTION
**Pull request by iarna**
[CHANGELOG](https://github.com/npm/npm/blob/release-next/CHANGELOG.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,54 @@
+### v4.1.2 (2017-01-12)
+
+We have a twee little release this week as we come back from the holidays.
+
+#### 0.12 IS UNSUPPORTED NOW (really)
+
+After earlier jumping the gun a little, we can now officially remove 0.12
```
Comment (ashleygwilliams) [inline CHANGELOG.md]: move the word earlier, e.g. "after jump the gun a bit earlier, "

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,54 @@
...
+from our supported versions list.  This means we no longer run the test
```
Comment (ashleygwilliams) [inline CHANGELOG.md]: maybe link to https://github.com/npm/npm/blob/latest/.travis.yml#L9-L26

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,54 @@
...
+suite under 0.12 and if you run npm under 0.12 you'll receive a warning.
+
+* [`c7bbba8`](https://github.com/npm/npm/commit/c7bbba8744b62448103a1510c65d9751288abb5d)
+  Remove 0.12 from our supported versions list.
+  ([@iarna](https://github.com/iarna))
+
+#### WRITING TO SYMLINKED FILES
+
```
Comment (ashleygwilliams) [inline CHANGELOG.md]: might be a neat place to remind people that there is a `npm link`, and link to https://docs.npmjs.com/cli/link

Comment (iarna) [inline CHANGELOG.md]: this isn't related to that though.

File: CHANGELOG.md (hunk 1 above)
Comment (iarna) [inline CHANGELOG.md]: Actually maybe I'll just delete "earlier" entirely.

File: CHANGELOG.md (hunk 2 above)
Comment (iarna) [inline CHANGELOG.md]: Down below the narrative is a link to the commit…

**Pull request merged**